styleguide/source/_patterns/01-molecules/06-components/00-figure/_figure.scss

```diff
@@ -32,6 +32,7 @@
 .figure_description {
   color: $black-55;
   margin: 15px 0 0;
+  font-weight: 500;
```

Comment:
@aCyborg it looks like the `.figure_description` text in the comp is suppose to be set to a regular weight while the link is semibold.